src/main/java/org/oasis_eu/portal/config/OasisPortalSecurity.java

```diff
@@ -52,11 +54,18 @@ protected void configure(HttpSecurity http) throws Exception {
                 .logout().logoutRequestMatcher(new AntPathRequestMatcher("/logout")).logoutSuccessHandler(logoutHandler()).and()
                 .exceptionHandling().authenticationEntryPoint(authenticationEntryPoint()).and()
                 .authorizeRequests()
+                .antMatchers("/my/api/organization/import").permitAll()
                 .antMatchers("/my/**").authenticated()
                 .anyRequest().permitAll().and()
+                .csrf().ignoringAntMatchers("/my/api/organization/import").and()
                 .addFilterBefore(oasisAuthenticationFilter(), AbstractPreAuthenticatedProcessingFilter.class);
         }
         http
             .addFilterAfter(oasisExceptionTranslationFilter(authenticationEntryPoint()), ExceptionTranslationFilter.class);
     }
+
+    @Bean
+    public PasswordEncoder passwordEncoder() {
```

Comment:
For such a single specific use, better instantiate it directly in the service (what's more, dependency injection does not bring a lot here)

Reply:
OK